### ISSUE TITLE
Consider/resolve safety issues in using subprocesses and uvicorn module

### ISSUE THREAD

**Issue by davidverweij** [ideafast/middleware-services#38]
`Flake8` throws the following errors:
```shell
cli.py:1:1: S404 Consider possible security implications associated with subprocess module.
cli.py:12:1: S602 subprocess call with shell=True identified, security issue.
cli.py:34:1: S104 Possible binding to all interfaces.
```
As I didn't want to change the setup without discussion, I've excluded these lines from linting by adding 
```python
# noqa
```

We should address/discuss these concerns

**Comment by jawrainey** [comment]
Good point 👍🏼  I assume this "security issue" is because: (i) any command could then be run via shell (`rm -rf ~/`) and; (ii)  that such commands could then invoke other scripts or provide access to unauthorised users in the shell.

I think `shell=True` can be removed from the subprocess call [here](https://github.com/ideafast/middleware-services/blob/master/cli.py#L11) since the `command` argument is within a list.

@davidverweij -- either way, might be worth installing and running [bandit](https://pypi.org/project/bandit/) or such via pre-commit to identify this and potentially other security issues within the codebase as development progresses?
